packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -53,6 +54,20 @@ typedef SemanticsActionHandler = void Function(Object? args);
 /// Used by [SemanticsOwner.onSemanticsUpdate].
 typedef SemanticsUpdateCallback = void Function(ui.SemanticsUpdate update);
 
+/// Signature for the [SemanticsConfiguration.childConfigsDelegate].
```

Comment:
here and elsewhere: In line with our style guide we probably shouldn't be using the "configs" abbreviation and instead call this "configuration" everywhere.